app/src/main/java/chat/rocket/android/webview/oauth/ui/OauthWebViewActivity.kt

```diff
@@ -56,6 +57,11 @@ class OauthWebViewActivity : AppCompatActivity() {
         }
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        CookieManager.getInstance().removeAllCookies(null)
```

Comment:
Wouldn't be better to clean it on `onCreate` before setting up the `WebView`, because if for some reason we crash after the user authenticated, while parsing the response, we will never cleanup the cookies.

Reply:
@luciofm Changed.